release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3598,4 +3601,4 @@ $ oc adm release info 4.12.15 --pullspecs
 [id="ocp-4-12-15-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

Comment:
I didn't add this. It was already there but because of a rebase its saying I added this line.